xxk3990/asti-memories
# Commenting and liking added

To test:

Download and run the code. Run the react. Create a memory. Go into the db and verify it was added. Once the memory is added, it should take you to the list of memories with the one you just created added to it and all the necessary data showing. Try to like the post you added and add a comment to it. The number of likes should go up and the comment should appear in the comments list.  Add a second comment to the same post to verify both are appearing. That's all that has been implemented so far.
